tests/pytorch_tests/model_tests/test_feature_models_runner.py

```diff
@@ -137,42 +137,54 @@ def test_bn_folding(self):
         """
         This test checks the BatchNorm folding feature.
         """
-        assert False
 
         def batch_norm_wrapper(channels):
             return partial(nn.functional.batch_norm,
-                           running_mean=torch.zeros(channels, device='cuda'),
-                           running_var=torch.ones(channels, device='cuda'))
+                           running_mean=0+torch.randn(channels, device='cuda'),
```

Comment:
you'll probably need to select the device accordingly, because tests are ran on CPU in github